### PR TITLE
Feat : Rating, SnackBar Component

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
### Change Log

Added Rating and SnackBar Components

### CODE REVIEW

**Review by saifudinbadani — APPROVED** [review]
good to go.

File: src/components/rating.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+        <div class="ui-sideNav-item">
+          <a
+            href="./rating.html"
+            class="text-color-white gutter-small text-decoration-none"
+            >Rating</a
+          >
+        </div>
+        <div class="ui__sideNav__item">
+          <a
```
Comment (darshandamre) [inline src/components/rating.html]: i think in this class name you have used double underscores but in rest of the classes you have use single hyphens. same issue in rest of the files

File: src/components/rating.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            href="../../index.html"
+            class="text-color-white text-decoration-none"
+            >UI Docs</a
+          ></span
+        >
+        <i
+          class="fas fa-times-circle ui-sideNav-icon text-medium"
+          aria-hidden="true"
+          onClick="closeSideNav()"
+        ></i>
```
Comment (darshandamre) [inline src/components/rating.html]: this onclick can be all lowercase, it works this way too but better to have in lowercase to not to confuse with react onClick function. same thing in rest of the files.

File: css/components/rating.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.rating {
+  width: 50px;
+  height: 50px;
+  fill: var(--light-gray);
+}
```
Comment (ark2002) [inline css/components/rating.css]: should use relative sizing for anything above 10px

File: src/usage.html [hunk 4]
```diff
@@ -98,18 +98,24 @@
...
             >Navigation</a
           >
         </div>
         <div class="ui-sideNav-item">
-          <a href="#" class="text-color-white gutter-small text-decoration-none"
-            >Toast</a
+          <a
+            href="./snackBar.html"
+            class="text-color-white gutter-small text-decoration-none"
+            >Snackbar</a
```
Comment (darshandamre) [inline src/usage.html]: the links here don't work, you have given incorrect path

**Review by darshandamre — COMMENTED** [review]
some minor issues, rest everything is fine

**Review by ark2002 — CHANGES_REQUESTED** [review]
heck the changes rest looks fine

**Comment by SRUJANA-PENUGONDA13** [comment]
Thank You @ark2002 @darshandamre @saifudinbadani

**Pull request merged**